arch/arm/core/aarch64/Kconfig

```diff
@@ -58,6 +58,12 @@ config IPM_CONSOLE_STACK_SIZE
 
 if CPU_CORTEX_A
 
+config ARMV8_A_NS
+	bool "ARMv8-A Normal World(None Secure world of Trustzonel)"
```

Comment:
typos: `ARMv8-A Normal World (Non-Secure world of Trustzone)`

Reply:
fix in next version